README.md

```diff
@@ -40,12 +40,22 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 - [ ] Explain the difference between `Relational Databases` and `SQL`.
 
+Relational Databases store data in tables. A table has a collection of records and fields. It is also where the resources are stored. 
+
+SQL stands for Structured Query Language. It manages CRUD data, databases, and it's objects and the server. 
+
 - [ ] Why do tables need a `primary key`?
 
+Because the primary key uniquely identifies each row.
+
 - [ ] What is the name given to a table column that references the primary key on another table.
 
+Foreign Key
+
 - [ ] What do we need in order to have a _many to many_ relationship between two tables.
 
+We would need records in one table to associate with records in another table.
+
```

Comment:
A many-to-many relationship occurs when multiple records in a table are associated with multiple records in another table. For example, a many-to-many relationship exists between customers and products: customers can purchase various products, and products can be purchased by many customers.